pkg/config/config.go

```diff
@@ -1266,18 +1278,41 @@ func (c *RuntimeConfig) ValidateRuntimes() error {
 }
 
 func (c *RuntimeConfig) initializeRuntimeFeatures() {
-	for _, handler := range c.Runtimes {
+	for name, handler := range c.Runtimes {
 		// If this returns an error, we just ignore it and assume the features sub-command is
 		// not supported by the runtime.
 		output, err := cmdrunner.Command(handler.RuntimePath, "features").CombinedOutput()
 		if err != nil {
 			logrus.Errorf("Getting %s OCI runtime features failed: %s: %v", handler.RuntimePath, output, err)
 			continue
 		}
-		// Ignore errors to Unmarshal too, we can't populate it.
-		if err := json.Unmarshal(output, &handler.features); err != nil {
-			logrus.Errorf("Unmarshalling OCI features failed: %s", err)
+
+		// Ignore error if we can't load runtime features.
+		if err := handler.LoadRuntimeFeatures(output); err != nil {
+			logrus.Errorf("Unable to load OCI features for runtime handler %q: %v", name, err)
+			continue
```

Comment:
**To the reviewer:**

This was originally missing. I think it was a small bug.

Reply:
If we're ignoring the error here, should it be `logrus.Warnf` not `Errorf`?

Reply:
I can change these to warnings. However, an error carries much more weight than a warning when we want to bring something to the user's attention, especially during initialisation. Perhaps that is why this was originally done that way.